conda_build/build.py

```diff
@@ -336,6 +339,12 @@ def create_info_files(m, files, config, prefix):
     write_info_json(m, config, mode_dict)
     write_about_json(m, config)
 
+    if str(m.get_value('build/noarch')).lower() == "python":
```

Comment:
Since this expression is a bit ugly, and you use it multiple times, I recommend turning it into a function.